diff --color -ru hyprsunset-0.2.0/src/Hyprsunset.hpp hyprsunset-0.2.0-patched/src/Hyprsunset.hpp
--- hyprsunset-0.2.0/src/Hyprsunset.hpp	2025-03-24 07:36:03.000000000 -0400
+++ hyprsunset-0.2.0-patched/src/Hyprsunset.hpp	2025-05-01 19:39:16.367816750 -0400
@@ -1,7 +1,7 @@
 #include <iostream>
 #include <cmath>
 #include <algorithm>
-#include <sys/signal.h>
+#include <signal.h>
 #include <wayland-client.h>
 #include <vector>
 #include "protocols/hyprland-ctm-control-v1.hpp"
